app/assets/javascripts/edit/id.js.erb

```diff
@@ -36,4 +36,10 @@ $(function () {
   if (idData.gpx) params.set("gpx", idData.gpx);
 
   id.attr("src", idData.url + "#" + params);
+
+  id.ready(function () {
+    if (!this.contentWindow.location.href.startsWith(idData.url)) {
```

Comment:
I get a `Uncaught TypeError: this.contentWindow is undefined` error every time I regularly start an iD editing session. Is that expected? Would it not be better to skip the check if everything is loaded correctly, maybe like this:

```js
if (this.contentWindow && !this.contentWindow.location.href.startsWith(idData.url)) {
```

Reply:
I'm not sure if this fix ever worked. I've seen double header from #5807 with this PR merged.